### DISCUSSION TITLE
Could someone create a example of audio processing each frequency and how loud is each frequency of the play back?

### DISCUSSION THREAD

**Discussion by War3Evo** [RuudErmers/ClapForDelphi#5] [General]
Hi!

Can someone please create a sample plugin that processes just the frequencies of sound including how loud each frequency of sound?

I'm not trying to process midi or create midi instrument.  Just wish to process audio, Thank you!

**Comment by RuudErmers** [comment]
You should look at the DelphiAsioVST library. It has tons of Dsp functions. Not trivial though!
(https://github.com/RuudErmers/DelphiASIOVST-64-bit-Examples-2018/tree/master/DelphiASIOVST-v1.4)
